keyboards/blank_tehnologii/manibus/keymaps/samurai/keymap.c

```diff
@@ -0,0 +1,222 @@
+/* Copyright 2021 SamuraiKek
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+
+#define _QWERTY 0
+#define _LOWER 1
+#define _UPPER 2
+#define _MOUSE 3
+#define _MEDIA 4
```

Comment:
We highly recommend using enums for the layer naming, as it is easier to maintain long term. 

```suggestion
enum custom_layers {
    _QWERTY,
    _LOWER,
    _UPPER,
    _MOUSE,
    _MEDIA,  
);
```